src/Essentials/src/PhoneDialer/PhoneDialer.shared.cs

```diff
@@ -50,6 +50,10 @@ public static class PhoneDialer
 		public static void Open(string number)
 			=> Current.Open(number);
 
+		/// <summary>
+		/// Provides the default implementation for static usage of this API.
+		/// This is the same as <see cref="Default"/>.
```

Comment:
Maybe not this one as Default needs to be removed as it was a bad merge. We can probably obsolete/hide it now already.